icons/modrinth.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Modrinth</title><path d="M 4.629 14.971 A 7.875 7.875 0 0 1 4.051 12 c 0 -4.37 3.553 -7.917 7.93 -7.917 c 0.135 0 0.27 0.003 0.405 0.01 l -0.38 2.151 h -0.025 a 5.763 5.763 0 0 0 -5.452 7.632 l 2.346 -1.352 l -0.5 -1.724 l 1.923 -2.16 l 2.644 -0.72 l 0.962 0.96 l -1.443 1.44 l -0.961 0.24 l -0.721 0.96 l 0.269 1.018 l 0.748 0.741 l 1.193 -0.143 l 0.689 -0.712 l 1.97 -0.527 l 0.352 1.312 l -1.946 1.926 l -2.835 0.583 l -1.098 -1.141 l -2.35 1.354 a 5.755 5.755 0 0 0 4.737 1.801 l 0.566 2.11 a 7.92 7.92 0 0 1 -7.213 -2.809 l -1.686 0.972 a 9.861 9.861 0 0 0 16.694 -1.753 l 2.042 0.742 A 12.027 12.027 0 0 1 11.981 24 C 5.671 24 0.49 19.135 0 12.96 h 2.172 a 9.78 9.78 0 0 0 0.776 2.98 l 1.681 -0.969 Z m 16.51 -6.608 a 9.863 9.863 0 0 0 -9.158 -6.203 c -5.115 0 -9.326 3.9 -9.809 8.88 H 0 C 0.49 4.865 5.671 0 11.981 0 C 18.614 0 24 5.377 24 12 c 0 1.104 -0.15 2.173 -0.43 3.19 l -2.042 -0.742 a 9.833 9.833 0 0 0 0.22 -3.763 l -1.879 0.502 c 0.027 0.268 0.041 0.539 0.041 0.813 c 0 3.318 -2.048 6.162 -4.95 7.339 l -0.566 -2.111 A 5.758 5.758 0 0 0 17.746 12 a 5.764 5.764 0 0 0 -3.818 -5.419 l 0.38 -2.151 a 7.955 7.955 0 0 1 4.955 4.435 l 1.876 -0.502 Z"/></svg>
```

Comment:
The SVG path data does not seem to be correctly optimized using SVGO/SVGOMG because there are at least too many whitespaces.